vector/src/main/res/menu/vector_home_room_settings.xml

```diff
@@ -1,39 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
-    <group android:id="@+id/active_room_actions">
 
+    <group android:id="@+id/add_shortcut_actions">
         <item
-            android:id="@+id/ic_action_select_notifications"
-            android:icon="@drawable/ic_material_done"
-            android:title="@string/room_settings_notifications" />
-
-        <!-- item no more home nor favorite so dont allow these actions
-            android:id="@+id/ic_action_select_fav"
-            android:icon="@drawable/ic_material_done"
-            android:title="@string/room_settings_favourite" />
+            android:id="@+id/ic_action_add_homescreen_shortcut"
+            android:title="Créer un raccourci" />
```

Comment:
are we sure we do not want to use i18n tool in this project?

Reply:
I don't understand what is i18n tool ?

Reply:
i18n is internationalisation. 
Here, "Créer un raccourci" is written in the code instead of a resource file (string.xml). I do not think this is a good practice.